platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -2047,9 +2047,15 @@
             var exists;
 
             if ($scope.advancedSettings !== null && $scope.lookup !== undefined && $scope.lookup.lookupName !== undefined) {
-                LoadingModal.open();
                 $scope.validateLookupName($scope.lookup.lookupName);
-                LoadingModal.close();
+            }
+        });
+
+
+        $rootScope.$on('$stateChangeStart', function(event, toState, toParams, fromState, fromParams) {
```

Comment:
@jredlarski @MichalKus Do we really need catch every change of state in this place?